popup.js

```diff
@@ -1345,6 +1345,7 @@ function buildTree(data, action, sendid) {
                 submitstatus.pop();
             });
         } else {
+            //console.log("Envoi rqv3 add-photo",data,posturl);
```

Comment:
I guess you'd want to remove that

Reply:
certainly, it is only useful for development and debugging